week1/homework/src/server.js

```diff
@@ -6,10 +6,33 @@ const http = require('http');
  * will fail.
  */
 function createServer(port) {
+  // TODO: Write your homework code here
   let state = 10;
-
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    function responseWrite(statusCode, result = {}) {
+      response.writeHead(statusCode, { 'Content-Type': 'application/json' });
+      response.write(JSON.stringify(result));
+    }
+
+    switch (request.url) {
+      default:
+        responseWrite(404, { error: 'Not found' });
+        break;
```

Comment:
for a best practice and more legibility I would go for keeping this `default` case at the end of my code.

Reply:
owww ok I will change it right away,